src/app/api/voice-logs/create/route.ts

```
@@ -0,0 +1,463 @@
+    let error: string | undefined
+
+    switch (intent) {
+      case 'irrigation':
+        const irrigationResult = await saveIrrigationLog(
+          supabase,
+          farmId,
+          data as IrrigationLogData,
+          conversationId,
+          transcript,
+          confidence
+        )
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)